### DISCUSSION TITLE
How do I add a button to 'accounts/context' through ‘vscode Plugin API’?

### DISCUSSION THREAD

**Discussion by MrZmcH** [microsoft/vscode-discussions#223] [Extension Development QnA] [answered]
![eyFSx](https://user-images.githubusercontent.com/45864434/200457642-af48e53f-d7ad-4331-8438-f1241ab931a9.png)
 I am going to develop a plugin to add one or more buttons in the above position.
Based on the documentation I found, my 'package.json' looks like this：
![swbDG](https://user-images.githubusercontent.com/45864434/200457649-cbf40baa-0347-4b13-9c63-78c70d3d0226.png)
After running F5, it did not achieve the desired effect. Who has such experience?
 it's not mentioned in the official api, but I saw it written in this way, so I asked, how do I implement it?

**Comment by gjsjohnmurray** [comment, marked as answer]
`account/context` is not listed at https://code.visualstudio.com/api/references/contribution-points#contributes.menus so I don't think it can be done.
